library_generation/repo_generation/repo_level_post_processor.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import subprocess
+from pathlib import Path
+
+
+def repo_level_post_process(
```

Comment:
Can we write unit tests for new Python scripts if possible?

Reply:
Can I add unit tests after #2342 is merged since the python unit test infrastructure is setup in that PR.

Also, I still need another PR to refactor `generate_repo.py` and I'll include more unit tests in that PR.

Reply:
@JoeWang1127 I added a few units for `ClientInputs` in #2342